### PR TITLE
feat: add rotational cipher exercise

### PR DESCRIPTION
**Pull request by vaeng**
According to https://gist.github.com/ErikSchierboom/2f121db2bc55003129408edcd19183dc, this exercise is the second most popular (among other tracks) exercise, that has not yet been made available on the cpp track.

Let's change that.

### CODE REVIEW

File: exercises/practice/rotational-cipher/.meta/example.cpp [hunk 1]
```diff
@@ -0,0 +1,22 @@
+#include "rotational_cipher.h"
+
+#include <sstream>
+
+namespace rotational_cipher {
+    std::string  rotate(const std::string& text, int shiftKey) {
+
+        std::ostringstream encoded_text;
```
Comment (siebenschlaefer) [inline exercises/practice/rotational-cipher/.meta/example.cpp:8]: style nitpick: "lowerCamelCase" for the parameter but "snake_case" for the variable. Is that intentional?

File: exercises/practice/rotational-cipher/.meta/example.cpp [hunk 2]
```diff
@@ -0,0 +1,22 @@
+#include "rotational_cipher.h"
+
+#include <sstream>
+
+namespace rotational_cipher {
+    std::string  rotate(const std::string& text, int shiftKey) {
```
Comment (siebenschlaefer) [inline exercises/practice/rotational-cipher/.meta/example.cpp:6]: style nitpick: are the two spaces between return type and function name intentional? (in the .h file, too)

File: config.json [hunk 3]
```diff
@@ -1036,6 +1036,15 @@
...
         ]
+      },
+      {
+        "slug": "rotational-cipher",
+        "name": "Rotational Cipher",
+        "uuid": "21d02535-4ab6-455d-88e1-08ec3b578894",
+        "practices": [],
+        "prerequisites": [],
+        "difficulty": 3,
+        "topics": []
```
Comment (siebenschlaefer) [inline config.json:1047]: I think "strings" and "loops" would be good topics.
Maybe "math", too, because of the modulo arithmetic.

**Review by siebenschlaefer — APPROVED** [review]
Great! No issues or complaints, just two small nitpicks and maybe some topics.

File: config.json (hunk 3 above)
Comment (vaeng) [reply config.json:1047]: Every concept that is added in the "practices" field, makes the syllabus tree a bit bigger. Loops is quite big already, but we could add loops as a prereq and add it to the strings concept, so it is only available if people finished loops?

Comment (siebenschlaefer) [reply config.json:1051]: Oh, I didn't realize that. I'll try to remember that.

**Pull request merged**